host/src/pack/dpkg/etc/rc.civ.post

```diff
@@ -65,6 +65,10 @@ balloon_civ
 
 /opt/lg/bin/LG_B1_Client_clipboard guestClipboard:enable=true &
 
-flock -n -x /var/lock/civ_startapp.lock pause_civ ||
+exec {CIV_STARTAPP_LOCK_FD}</var/lock/civ_startapp.lock
```

Comment:
sleep 20
touch /var/lock/civ_startapp.lock

added two lines above in customer's releaes